scripts/ansible/roles/linux/openenclave/vars/main.yml

```diff
@@ -24,3 +24,9 @@ validation_binaries:
   - "/usr/bin/pkg-config"
   - "/usr/bin/git"
   - "/usr/bin/svn"
+
+dcap_package_url: https://packages.microsoft.com/ubuntu/20.04/prod/pool/main/a/az-dcap-client/az-dcap-client_1.12.3_amd64.deb
```

Comment:
This var can be removed now that we can use the official release.
If the openssl tasks are no longer needed as well, then `ssl_package_url` and `package_dest_path` can be removed too.

Reply:
Done